build/bazel/remote/asset/v1/BUILD

```diff
@@ -12,9 +12,9 @@ proto_library(
         "//build/bazel/remote/execution/v2:remote_execution_proto",
         "@com_google_protobuf//:duration_proto",
         "@com_google_protobuf//:timestamp_proto",
-        "@googleapis//:google_api_annotations_proto",
-        "@googleapis//:google_api_http_proto",
-        "@googleapis//:google_rpc_status_proto",
+        "@googleapis//google/api:annotations_proto",
```

Comment:
Nice! I hope that this also makes my life easier on the Buildbarn side. I currently apply this local change to be able to build these Protobufs.

https://github.com/buildbarn/bb-storage/blob/master/patches/com_github_bazelbuild_remote_apis/golang.diff

Reply:
Hmm... And the build still fails. Maybe you need to upgrade `http_archive(name = "googleapis")` in `WORKSPACE` as well?

Reply:
I don't understand the error :-(

Reply:
I works for me locally.

Reply:
Could it be a bazel version difference?

Reply:
I use 3.7.2.

Reply:
The CI uses basilisk and Bazel version 3.7.2